target/linux/mvebu/image/cortexa9.mk

```diff
@@ -32,8 +32,8 @@ define Device/cznic_turris-omnia
   DEVICE_VENDOR := CZ.NIC
   DEVICE_MODEL := Turris Omnia
   KERNEL_INSTALL := 1
-  KERNEL := kernel-bin
-  KERNEL_INITRAMFS := kernel-bin
+  KERNEL := kernel-bin | append-dtb
+  DEVICE_DTS := armada-385-turris-omnia
```

Comment:
This line is not necessary, DEVICE_DTS is constructed from SOC and the device name:
https://github.com/openwrt/openwrt/blob/master/target/linux/mvebu/image/Makefile#L101

Reply:
No idea why but previously it didn't work. It was always looking for `armada--turris-omnia` instead of this. There might have been fix in the meantime to this so.. line removed.